src/libraries/Microsoft.PowerFx.Core/Types/DTypeSpecParser.cs

```diff
@@ -46,7 +47,7 @@ public static bool TryParse(DTypeSpecLexer lexer, out DType type)
                 return true;
             }
 
-            var typeIdx = _typeEncodings.IndexOf(token);
+            var typeIdx = _typeEncodings.IndexOf(token, StringComparison.Ordinal);
```

Comment:
> StringComparison.Ordinal); [](http://example.com/codeflow?start=55&length=27)

This is excellent but we should enable globalization .Net analyzers at the same time

Reply:
+1 for enabling globalization .Net analyzers - it would probably have caught these and others.  But out of scope for this PR.